### ISSUE TITLE
Violence Category

### ISSUE THREAD

**Issue by H7-25** [infinitered/nsfwjs#574]
Why not include also violence category ? Any available model to check about it ?

**Comment by GantMan** [comment]
Good call.   We're not actively adding features at this time, but this would def be a good feature.

**Comment by hosnipogi** [comment]
How does one add another category to this library, like the violence category for example?
